app/views/nic/_base_form.html.erb

```diff
@@ -63,17 +63,17 @@
 
 <% if SETTINGS[:unattended] %>
   <%= checkbox_f f, :managed,
-                 :label_help => _("Should this interface be managed via DHCP and DNS smart proxy and should it be configured during provisioning?"),
+                 :label_help => _("If the Managed flag is enabled, external services such as DHCP, DNS, and TFTP will be configured according to the information provided.<br/>The services to be configured is determined by the subnet and domain that are selected for this interface. If you want to manage<br/> only a subset of these services, change the %{subnet} and %{domain} proxies configuration. When a proxy is set to 'None', the corresponding<br/> service configuration will be skipped.<br/><br/>If the Managed flag is disabled, none of the services will be configured for this interface, even if it is configured for the subnet and domain.<br/><br/>The Managed flag also determines whether the interface should be configured during host provisioning, so that the configuration remains after reboot.") % { :subnet => link_to(_('subnet'), subnets_path, :target => :blank), :domain => link_to(_('domain'), domains_path, :target => :blank)},
```

Comment:
`The services to be configured *are* determined`.
In my opinion, the part about `When a proxy is set to 'None', the corresponding<br/> service configuration will be skipped.<br/>`  doesn't really add anything, it's to be expected.